src/intervals/rationalize.jl

```diff
@@ -50,4 +50,5 @@ function old_rationalize{T<:Integer}(::Type{T}, x::AbstractFloat; tol::Real=eps(
     return convert(T,a)//convert(T,b)
 end
 
-old_rationalize(x::Union(Float64,Float32); tol::Real=eps(x)) = old_rationalize(Int, x, tol=tol)
+old_rationalize(x::Float64; tol::Real=eps(x)) = old_rationalize(Int, x, tol=tol)
+old_rationalize(x::Float32; tol::Real=eps(x)) = old_rationalize(Int, x, tol=tol)
```

Comment:
I agree. However, `Union(..)` is deprecated in favor `Union{..}`, which has problems in version 0.3...